src/index.js

```diff
@@ -191,7 +202,29 @@ class KadDHT {
         (cb) => this._putLocal(key, rec, cb),
         (cb) => this.getClosestPeers(key, cb),
         (peers, cb) => each(peers, (peer, cb) => {
```

Comment:
I forgot to get rid of the each in my diff, it needs to get removed. It can just be:
```suggestion
        (peers, cb) => {
```

Reply:
Somehow missed that.  Meant to remove it.